lib/navigation/ui/website_navigation.dart

```diff
@@ -68,18 +67,9 @@ class _WebsiteNavigationState extends State<WebsiteNavigation> {
                 ...widget.actions
                     .map(
                       (action) => ContextMenuItem(
-                        title: action.title,
```

Comment:
As I remember, there was a reason to add this default onTap callback. Can we make it the same here?

Reply:
Because, it had error.

<img width="539" alt="Ekran Resmi 2022-02-27 00 15 00" src="https://user-images.githubusercontent.com/13748518/155859315-df41ca24-c589-46be-98c4-c90dd1dc11c2.png">